www/js/app.js

```diff
@@ -1017,6 +1020,10 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
         // but we should not do this when opening the landing page (or else one of the Unit Tests fails, at least on Chrome 58)
         if (!params.isLandingPage) document.getElementById('articleContent').contentWindow.focus();
 
+        // Early return if article's title not equal current url's title
+        var matchUrl = dirEntry['namespace'] + "/" + dirEntry["url"];
+        if(matchUrl !== curTitle) return;
```

Comment:
It would be worth adding a `console.debug` here so that this kind of (rare) case could be easily detected and understood